helm/overrides/services.yaml

```diff
@@ -21,7 +21,7 @@ elasticsearch:
   persistence:
     enabled: false
 
-  replicas: 1
+  replicas: 2
```

Comment:
We don't want the default to spin up 2 replicas, as this takes up large amounts of resources. Please change this back to 1, add a note about scaling for testing.